src/detect-content.c

```diff
@@ -405,6 +411,12 @@ _Bool DetectContentPMATCHValidateCallback(const Signature *s)
                     cd->content_len, cd->offset, right_edge, max_right_edge);
             return FALSE;
         }
+	if (distance_size > (max_right_edge_i - content_total_length)) {
```

Comment:
Formatting is a bit odd when I open the file in my editor. Did you use tabs instead of spaces?
![Screenshot from 2019-07-23 21-39-21](https://user-images.githubusercontent.com/32410611/61742068-93eb1b80-ad92-11e9-9b3d-2f83c2584f56.png)

Reply:
Yes, that appears to be the case. fixing.